nautobot/dcim/tests/test_forms.py

```diff
@@ -343,9 +343,20 @@ def setUpTestData(cls):
 
     def test_interface_form_clean_vlan_location_fail(self):
         """Assert that form validation fails when no matching locations are associated to tagged VLAN"""
-        self.vlan.locations.set(list(Location.objects.exclude(pk=self.device.location.pk))[:2])
+        location = self.device.location
+        location_ids = [ancestor.id for ancestor in location.ancestors()]
```

Comment:
Do we need an `include_self=True` here?

Reply:
Yeah I believe so thanks for catching that